src/pybind/mgr/devicehealth/module.py

```diff
@@ -4,7 +4,7 @@
 
 import errno
 import json
-from mgr_module import MgrModule, CommandResult, CLIRequiresDB, CLICommand, Option
```

Comment:
Don't remove `CLICommand` and check if `CLIWriteCommand` is required

Reply:
Okay 👍

Reply:
`CLIWriteCommand` is not required so removing it from import.